src/common/ceph_crypto.cc

```diff
@@ -83,6 +84,7 @@ void ceph::crypto::shutdown()
   assert(crypto_refs > 0);
   if (--crypto_refs == 0) {
     NSS_ShutdownContext(crypto_context);
+    PR_Cleanup();
```

Comment:
Library users are not guaranteed to have ```g_ceph_context``` -- i.e. QEMU could hot-add and hot-remove RBD-backed drives in any order. Is this something that should only be invoked on Ceph daemons and applications?